packages/command-palette/src/commands/use-single-site-commands.tsx

```diff
@@ -0,0 +1,927 @@
+			callback: commandNavigation( '/wp-admin/media-new.php' ),
+			capability: SiteCapabilities.UPLOAD_FILES,
+			icon: mediaIcon,
+		},
+		{
+			name: 'managePages',
+			label: __( 'Manage pages', __i18n_text_domain__ ),
+			searchLabel: [
+				_x( 'manage pages', 'Keyword for the Manage pages command', __i18n_text_domain__ ),
+				_x( 'edit pages', 'Keyword for the Manage pages command', __i18n_text_domain__ ),
+				_x( 'delete pages', 'Keyword for the Manage pages command', __i18n_text_domain__ ),
+			].join( ' ' ),
+			callback: commandNavigation( '/wp-admin/edit.php?post_type=page' ),
+			capability: SiteCapabilities.EDIT_PAGES,
+			icon: editIcon,
+		},
+		{
+			name: 'addNewPage',
+			label: __( 'Add new page', __i18n_text_domain__ ),
+			searchLabel: [
+				_x( 'add new page', 'Keyword for the Add new page command', __i18n_text_domain__ ),
+				_x( 'create page', 'Keyword for the Add new page command', __i18n_text_domain__ ),
+				_x( 'write page', 'Keyword for the Add new page command', __i18n_text_domain__ ),
+			].join( ' ' ),
+			context: [ '/pages' ],
+			callback: commandNavigation( '/wp-admin/post-new.php?post_type=page' ),
+			capability: SiteCapabilities.EDIT_PAGES,
+			icon: plusIcon,
+		},
+		{
+			name: 'manageComments',
+			label: __( 'Manage comments', __i18n_text_domain__ ),
+			searchLabel: [
+				_x( 'manage comments', 'Keyword for the Manage comments command', __i18n_text_domain__ ),
+				_x( 'edit comments', 'Keyword for the Manage comments command', __i18n_text_domain__ ),
+				_x( 'delete comments', 'Keyword for the Manage comments command', __i18n_text_domain__ ),
+			].join( ' ' ),
+			callback: commandNavigation( '/wp-admin/edit-comments.php' ),
+			capability: SiteCapabilities.MODERATE_COMMENTS,
+			icon: postCommentsIcon,
+		},
+		{
+			name: 'manageThemes',
+			label: __( 'Manage themes', __i18n_text_domain__ ),
+			searchLabel: [
+				_x( 'manage themes', 'Keyword for the Manage themes command', __i18n_text_domain__ ),
+				_x( 'activate theme', 'Keyword for the Manage themes command', __i18n_text_domain__ ),
+				_x( 'install theme', 'Keyword for the Manage themes command', __i18n_text_domain__ ),
+				_x( 'delete theme', 'Keyword for the Manage themes command', __i18n_text_domain__ ),
+			].join( ' ' ),
+			callback: commandNavigation( '/wp-admin/themes.php' ),
+			capability: SiteCapabilities.EDIT_THEME_OPTIONS,
+			filterP2: true,
+			icon: brushIcon,
+		},
+		{
+			name: 'installTheme',
+			label: __( 'Install theme', __i18n_text_domain__ ),
+			searchLabel: [
+				_x( 'install theme', 'Keyword for the Install theme command', __i18n_text_domain__ ),
+				_x( 'add theme', 'Keyword for the Install theme command', __i18n_text_domain__ ),
+				_x( 'upload theme', 'Keyword for the Install theme command', __i18n_text_domain__ ),
+			].join( ' ' ),
+			callback: commandNavigation( '/wp-admin/theme-install.php' ),
+			capability: SiteCapabilities.EDIT_THEME_OPTIONS,
+			icon: brushIcon,
+		},
```

Comment:
The "Install theme" command should be available on Atomic sites only.

Reply:
It's now only available for Atomic sites.